VTEX - Subscriptions API (v3).json

```diff
@@ -2693,287 +3047,248 @@
-                "properties": {
-                    "begin": {
-                        "type": "string",
-                        "nullable": true,
-                        "description": "Subscription's beginning date with the format `yyyy-mm-ddThh:mm:ss`.",
-                        "example": "2022-06-10T00:00:00"
-                    },
-                    "end": {
-                        "type": "string",
-                        "nullable": true,
-                        "description": "Subscription's ending date with the format `yyyy-mm-ddThh:mm:ss`.",
-                        "example": "2023-06-10T00:00:00"
-                    }
-                },
-                "additionalProperties": false
-            },
-            "PlanThinRequest": {
-                "title": "PlanRequest",
-                "description": "Information about the plan.",
+                "description": "[Subscription plan](https://help.vtex.com/en/tutorial/subscription-plans-beta--5kczKRqHEsrs1tYtRcY8wR) object.",
                 "required": [
-                    "frequency",
                     "id",
-                    "purchaseDay"
+                    "frequency"
                 ],
-                "type": "object",
                 "properties": {
                     "id": {
                         "type": "string",
-                        "nullable": true,
-                        "description": "ID of the plan.",
-                        "example": "store.subscription"
+                        "description": "Subscription plan ID.",
+                        "example": "subscription.clothes"
                     },
                     "frequency": {
-                        "$ref": "#/components/schemas/FrequencyRequest"
+                        "required": [
+                            "interval",
+                            "periodicity"
+                        ],
+                        "type": "object",
+                        "description": "Subscription plan frequency.",
+                        "properties": {
+                            "periodicity": {
+                                "type": "string",
+                                "description": "Plan repurchase frequency, defines the subscription recurrence.",
+                                "enum": [
+                                    "DAILY",
+                                    "WEEKLY",
+                                    "MONTHLY",
+                                    "YEARLY"
+                                ],
+                                "example": "MONTHLY"
+                            },
+                            "interval": {
+                                "type": "integer",
+                                "description": "Time interval configured between subscription orders, which depends on the periodicity. For a `DAILY` periodicity, the value will be days, for `MONTHLY` it will be months, and so on.",
```

Comment:
```suggestion
                                "description": "Time interval configured between subscription orders, which depends on the periodicity. For a `DAILY` periodicity, the value will be measured in days, for `MONTHLY` it will be measured in months, and so on.",
```